### ISSUE TITLE
Regression in v5.1.0: Failed to decode request as JSON with firefox (was working in v5.0.3)

### ISSUE THREAD

**Issue by jlchereau** [webdriverio/webdriverio#3203]
#### Environment:

WebdriverIO version: 5.1.0
Mode: WDIO Testrunner
If WDIO Testrunner, running sync/async: sync
Node.js version: 11.6.0
NPM version: 6.5.0
Browser name and version: Firefox v64.0 (64-bit)
Platform name and version: Windows 10 v1809
Additional wdio packages used (if applicable): @wdio/selenium-standalone service

#### Describe the bug

With:

```
capabilities = [        {
  maxInstances: 1,
  browserName: 'firefox'
}],
```

all tests now trigger in webdriverio v5.1.0 (were working in v5.0.3)

```
[0-3] 2018-12-28T17:13:27.286Z WARN webdriver: Request failed due to Failed to decode request as JSON:
[0-3] 2018-12-28T17:13:27.288Z WARN webdriver: Request failed due to Failed to decode request as JSON:
[0-3] 2018-12-28T17:13:27.291Z WARN webdriver: Request failed due to Failed to decode request as JSON:
[0-3] 2018-12-28T17:13:27.293Z ERROR webdriver: Request failed due to Error: Failed to decode request as JSON:
    at Request._callback (.\node_modules\webdriver\src\request.js:95:27)
    at Request.self.callback (.\node_modules\request\request.js:185:22)
    at Request.emit (events.js:188:13)
    at Request.EventEmitter.emit (domain.js:441:20)
    at Request.<anonymous> (.\node_modules\request\request.js:1161:10)
    at Request.emit (events.js:188:13)
    at Request.EventEmitter.emit (domain.js:441:20)
    at IncomingMessage.<anonymous> (.\node_modules\request\request.js:1083:12)
    at Object.onceWrapper (events.js:276:13)
    at IncomingMessage.emit (events.js:193:15)
```

#### Expected behavior

As in v5.0.3

**Comment by cuki** [comment]
This is caused with #3190, as already mentioned in https://github.com/webdriverio/webdriverio/issues/3187#issuecomment-450195089 geckodriver doesn't like it when you send POST request without body.

Here is an example snippet for reproduction:
```js
const { remote } = require('webdriverio');

(async () => {
    const browser = await remote({
        logLevel: 'debug',
        path: '/',
        capabilities: {
            browserName: 'firefox'
        }
    });

    await browser.url('https://webdriver.io');
    await browser.url('https://google.com');
    await browser.back(); // POST request with no body

    const title = await browser.getTitle();
    console.log('Title was: ' + title);

    await browser.deleteSession();
})().catch((e) => console.error(e));
```

**Comment by christian-bromann** [comment]
This is fixed and will be released in `v5.1.1`
